### PR TITLE
3.0 updates

### PR DESCRIPTION
**Pull request by bicknellr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,47 +14,18 @@
...
   "version": "3.0.0-pre.20",
-  "resolutions": {
-    "inherits": "2.0.3",
-    "samsam": "1.1.3",
-    "supports-color": "3.1.2",
-    "type-detect": "1.0.0",
-    "@webcomponents/webcomponentsjs": "2.0.0-beta.2"
-  },
   "contributors": [
-    "T",
```
Comment (e111077) [inline package.json]: 😮

**Review by e111077 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,18 +1,77 @@
...
-IronMenuBehavior and IronMenubarBehavior, introduce multiple breaking changes.
-Please see the README for those behaviors for more detail.
+```js
+import {PolymerElement} from '@polymer/polymer/polymer-element.js';
+import {mixinBehaviors} from '@polymer/polymer/lib/legacy/class.js';
+import {html} from '@polymer/polymer/lib/utils/html-tag.js';
+
+import {IronMenuBehavior} from '@polymer/iron-menu-behavior/iron-menu-behavior.js';
+
+class SimpleMenu extends mixinBehaviors(IronMenuBehavior, PolymerElement) {
```
Comment (notwaldorf) [inline README.md:30]: Can you also have a "using this element in an html page" example after this? It's not clear how to use this element (what should go in the slots, etc)....

File: package.json (hunk 1 above)
Comment (notwaldorf) [reply package.json]: Not that it matters, but I've been moving this to "author" not "contributors"

**Review by notwaldorf — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (bicknellr) [reply package.json]: Updated.

File: README.md (hunk 2 above)
Comment (bicknellr) [reply README.md:30]: Added, PTAL.

**Review by notwaldorf — APPROVED** [review]

**Pull request merged**